### ISSUE TITLE
'Getting started'

### ISSUE THREAD

**Issue by j-gignoux** [de-jcup/eclipse-asciidoctor-editor#53]
Hello
I am very interested by the Asciidoctor eclipse plugin you provide, but...
How do you start creating a documentation project ? 
1. From your documentation it seems you only handle text files, not projects (ie there is no such thing as an 'asciidoctor perspective', do you confirm? nor an 'Asciidoctor project' in the eclipse 'file>new' menu?). Your videos are not clear on how you start from scratch (I mean, with a blank empty virgin workspace) since you already have files in there.
2. I am developing a big java project for which an important documentation must be setup and managed, and I thought keeping my asciidoc documents inside my java project (eg in a 'doc' package somewhere) could be a nice way to keep the doc in sync with the code, and to benefit of the code version management system (currently, mercurial, but probably moving to git soon). Does it make sense ?
3. Then, problem: when I create an asciidoc text file in my java project, the syntax highlighting apparently works ok (even though I am in the java perspective), but the preview does not work. I get this error 'Error opening file: No such file or directory' instead. Seems to be a very dum configuration problem (incomplete toolchain ? wrong output directory configuration? missing ruby library?), but since I never use ruby and dont know the basics about it, I cannot fix my problem without some very basic configuration information. Could you please tell me step by step how you setup your working environment once the plugin has been installed and download (ie, be more specific than just 'the editor has big dependencies' as stated in your wiki). You say that 'But those dependencies are packaged into additional plugins' - OK, which plugins ? can you provide a link to them?
Thanks for any help, I am impatient to start using this.
best regards,

**Comment by de-jcup** [comment]
Hello @j-gignoux ,

I just created 
https://github.com/de-jcup/eclipse-asciidoctor-editor/wiki/Getting-Started

To your points:
1. I made a new video: https://www.youtube.com/edit?o=U&video_id=Y2ODe5kqR1s
2.  It makes absolute sense to have documentation and code in same source repository. I am a fan of multiprojects (I use gradle) and would prefer own subproject(s) for documentation. But you could also do your documentation inside one project only. But I would not integrate it inside a package - or do you want to have your documentation inside your build results (jar, war, ear,...) too? I would use in a "one-project-setup" a subfolder outside the normal sources.
3. Separating parts
- Perspectives do not matter- the editor works everywhere
- In case of described failure: Move your mouse to he preview pane, press right mouse button and select "Reload" . If this does not work please use the toolbar icon 
![image](https://user-images.githubusercontent.com/11644753/40084596-e88f2588-5897-11e8-8a62-97054043bc16.png) to show generated document in your system browser. If its still not shown on browser the document is not generated
- If your Preview still does not work and you got still a:  'Error opening file: No such file or directory'.
  -  Please give me information about your system 
     - OS
     - Your hardware (just amount of RAM, Processors)
     - Version of Eclipse
     - Version of Java
     - Version of asciidoctor editor installed)
  - Please append a simple screenshot in your answer -showing the problem. 
- Ruby "problems"
  I updated the description about the plugins at https://github.com/de-jcup/eclipse-asciidoctor-editor/wiki/FAQ#initial-installation-slow-updates-fast maybe its more clear now.
  Long story short: Asciidoc editor must work out of the box! You must not install anything else then the editor (except Graphviz when you want to use plantuml or graphviz diagrams inside the editor)
